examples/org.eclipse.emfcloud.modelserver.example/src/main/java/org/eclipse/emfcloud/modelserver/example/client/ExampleModelServerClient.java

```diff
@@ -53,7 +53,7 @@ private ExampleModelServerClient() {}
 
    public static void main(final String[] args) {
       try (
-         ModelServerClient client = new ModelServerClient("http://localhost:8081/api/v1/",
+         ModelServerClientV2 client = new ModelServerClientV2("http://localhost:8081/api/v2/",
```

Comment:
Could you please have another look at this example client?

I have the following problems:
1. If subscription is closed due to server shutdown an Exception is thrown, previously the reason was logged.
2. The edit requests fail with `java.lang.RuntimeException: Could not parse 'data' field`
3. The undo/redo requests return `202: false Accepted`